packages/web-components/stencil.config.ts

```diff
@@ -35,7 +35,8 @@ export const config: Config = {
       esmLoaderPath: '../loader',
     },
     {
-      type: 'dist-custom-elements-bundle',
+      type: 'dist-custom-elements',
```

Comment:
This is the main change here that allows us to create [an optimized build](https://stenciljs.com/docs/custom-elements).

Reply:
Can you add that documentation link to the description of the PR? 🙏